### ISSUE TITLE
Decryptor not helpful for empty dens

### ISSUE THREAD

**Issue by DHYCIX** [MetaMask/vault-decryptor#2]
Throws following error on load (in browser debugger):

**Error in event handler for (unknown): TypeError: Cannot read property 'querySelectorAll' of null**

When hitting decrypt, shows following error (no matter if input fields filled or not):

**Uncaught ReferenceError: reason is not defined**

After that, nothing happens and no error text is shown on frontend.

Tried deploying the Decryptor on my own end but that gives the same errors.

--

I misplaced my seed and can't remember if I exported my private key, so any information regarding this would be beneficial to me, too. I still have the config data needed to decrypt the damn thing, but the Decryptor does unfortunately not work anymore. I'd be very glad to donate a bit of Ether for your troubles, if you'd share your wallet address with me.

If there will be no fix coming up, could you give me a hint where the default location for digitally saved seed words is or at least the default file name?

Thank you very much!

**Comment by danfinlay** [comment]
I haven’t changed this recently. Maybe try again on an updated browser?

 As for file location, try googling for your operating system and browser for where it stores extension data.

**Comment by DHYCIX** [comment]
Hi Dan,

Thanks for your quick answer! I'm really stressed out about this right now...

I'm using Windows 10 Pro. I tried with Chrome, Firefox, Opera and Edge on two different computers (IE didn't even bother loading the page); all browsers were up-to-date with all extensions disabled. I also disabled anti-virus, Windows Defender and the Windows Firewall, unfortunately to no avail.

Could it be possible, that the script is simply not working with current browsers anymore? Could you take a quick look and try for yourself? If it's a problem on my end, this might also give me a clou, or at least hope. Haha! :)

**Comment by danfinlay** [comment]
@DHYCIX the script seems to be working fine for me, but you may not be actually pasting the correct vault data. I'm curious what you're pasting, because in the same message you're asking where to find the data to paste.

What browser did you have MetaMask on previously? You would want to search google for "Windows 10 Pro [Your browser] extension data location".

**Comment by DHYCIX** [comment]
I followed the instructions [on this page](http://metamask.consensyssupport.happyfox.com/kb/article/22-how-to-copy-your-vault-data) and copy-pasted the data inside the big box where it says to put it. Then I enter my den's password (wrote this one down, so 100% sure it's right) and click the button. Then nothing happens. Reading through the decryptor's code, there should at least be an error message coming up (?) but instead the page does not change a bit, no matter how often I click the "Decrypt" button. Every time I do this, chrome displays the error message `Uncaught ReferenceError: reason is not defined`, though. That's why I thought, this one might be the culprit...

I used MetaMask on my Chrome browser. I also installed it on Firefox on a different computer, but I got locked out at some point, because I disabled the extension (on both computers)... Didn't bother logging in again, because I was sure I made a photo of my seed and wanted to do this another time... and now it's gone and I can't find it. :(

I also already dug through Chrome's local storage and extension files, thanks for the hint, though!
Is there something somewhere that I could use? I'm not really into this kind of stuff, but I managed to get a bunch of files containing all sorts of "keys", though I presume none of theme would be my private key? There is also interesting stuff in `AppData\Local\Google\Chrome\User Data\Default\Local Storage\leveldb\000208.ldb` where it says on one line `­//Y§x/@iDice/hi-there-you-can-exporthr-private-key-from-metamask` preceeded and followed by all kinds of garbled mess and number/letter combinations that could be my key...

**Comment by danfinlay** [comment]
Sorry this tool isn't more error-resilient.

It sound like you have a working version of MetaMask installed, and know the password? If that's the case, you can just export the seed phrase from the Settings view in the top-right menu.

I'll try to get this more robust soon, as long as you have those files backed up from a working MetaMask, it should be possible to decode it eventually, just maybe with a little more refining of this tool, or help from a programmer friend.

**Comment by DHYCIX** [comment]
I'm glad a solution for problems like these exists at all!

I have MetaMask installed on 2 different computers, but I'm locked out of both extensions (because I disable them at some point, I guess). I know the password to open my den, but the extension doesn't even ask me for it anymore, as if my den would not exist on any of these computers. When I click on the little fox head in Chrome, it instantly wants me to create a new den. I can't though, because I lost my 12-words seed and never exported my private key (or at least cannot remember that I did so).

**Comment by danfinlay** [comment]
Ah, it sounds like somehow the den got corrupted. It seems like sometimes browsers wipe it for no reason at all. This is one reason it's so important to always keep the seed phrase backed up.

Copying the vault from one of these MetaMasks is not likely to recover the vault. You need the vault from a working MetaMask. If your computer keeps regular backups, you would want a backup from a time when it worked.

**Comment by DHYCIX** [comment]
Hoooly shit...! Good to know!

Man, so many hours just to find out, there was nothing to restore with the Vault Decryptor after all!

I'll try a backup a few months back. Maybe I get lucky! Otherwise I'll give the Decryptor another try! Thank you very much for the help!

**Comment by dmaxwell23** [comment]
hello hopefully someone can help me I've searched tirelessly the past week for answers.  Before my chromebook crashed, I did a system backup and I have that file in .tini format.   I used both Brave and Chrome browsers before the crash so I need to locate the correct files on the chromeOS .tini recovery file.  I am now using a Windows 10 laptop.  I have had trouble renaming the .tini to a .tgz file to use in winzip.  Any suggestions on how I can find the vault data on the .tini chromeOS backup file?
